content/partials/versions/v1.1/core-features/_authentication_capabilities.textile

```diff
@@ -11,4 +11,4 @@ The following capability operations are available for API keys and issued tokens
 
 See a working "capabilities example":<%= JsBins.url_for('authentication/capabilities') %>, read "understanding capabilities and token security":/core-features/authentication#capabilities-explained above to get a more thorough overview of how capabilities can be used to secure your application along with working examples.
 
-While most of these capabilities need to be enabled for the resource you're using them with, as described in "resource names and wildcards":/core-features/authentication#wildcards above, there are exceptions. The @stats@ permission only does anything when attached to the wildcard resource @'*'@ (or a resource that contains that as a subset, such as @'[*]*'@), since stats are app-wide. The @channel-metadata@ permission works both ways. When associated with a specific channel or set of channels it allows you to use the "Channel Status API":/rest/channel-status#channel-status to request the status of that channel. When associated with the wildcard resource @'*'@ it takes on an additional meaning: as well as allowing channel status requests for all channels, it also allows you to "enumerate all active channels":/rest/channel-status#enumeration-rest
+While most of these capabilities need to be enabled for the resource you're using them with, as described in "resource names and wildcards":/core-features/authentication#wildcards above, there are exceptions. The @stats@ permission only does anything when attached to the wildcard resource @'*'@ (or a resource that contains that as a subset, such as @'[*]*'@), since stats are app-wide. The @channel-metadata@ permission works both ways. When associated with a specific channel or set of channels it allows you to use the "Channel Status API":/api/rest-sdk/channel-status#channel-status to request the status of that channel. When associated with the wildcard resource @'*'@ it takes on an additional meaning: as well as allowing channel status requests for all channels, it also allows you to "enumerate all active channels":/api/rest-sdk/channel-status#enumeration-rest
```

Comment:
I think `#enumeration-rest` is on the existing page, not the API reference.

Reply:
Well spotted! Fixed.